index.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+ <meta charset="UTF-8">
+
+ <title>Tic Tac Toe</title>
+ <link rel = "stylesheet" type = "text/css" href = "style.css">
+</head>
+<body>
+ <header id="header"><h1> Tic Tac Toe</h1></header>
+
+<div class="container" id = "board">
+ <div class = "row1">
+   <div class = "box" id = "box1"> </div>
+   <div class = "box" id = "box2"> </div>
+   <div class = "box" id = "box3"> </div>
+ </div>
+ <div class = "row2">
+   <div class = "box" id = "box4"> </div>
+   <div class = "box" id = "box5"> </div>
+   <div class = "box" id = "box6"> </div>
+ </div>
+ <div class = "row3">
```

Comment:
Instead of writing `class="box"` 9 times, you could write `class="row"` 3 times, and then select all the boxes with `$(".row div")`.

Also, I'd recommend not putting spaces around `=`: it can cause some browsers to break.